projects/Mallard/ios/Mallard/Info.plist

```diff
@@ -62,8 +62,6 @@
 			</dict>
 		</dict>
 	</dict>
-	<key>NSLocationWhenInUseUsageDescription</key>
```

Comment:
Can we double check this?

Reply:
https://github.com/facebook/react-native/issues/20879
No longer bundled in rn.